test/e2e/run_test.sh

```diff
@@ -272,8 +290,11 @@ if [[ -z "${SKIP_SETUP:-}" ]]; then
 fi
 
 if [[ -z "${SKIP_BUILD:-}" ]]; then
-    setup_kind_registry
-    build_and_push_images
+     setup_kind_registry
+   build_and_push_images
+   echo "Building kmeshctl CLI..."
+   make kmeshctl || { echo "Failed to build kmeshctl" >&2; exit 1; }
```

Comment:
I thought we don't compile kmeshctl? Already done in previous `build_and_push_images`?